drivers/gpio/Kconfig.ameba

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2024 Realtek Semiconductor Corp.
+# SPDX-License-Identifier: Apache-2.0
+
+config GPIO_AMEBA
+	bool "GPIO controller driver for Realtek Ameba series SoC"
+	default y
+	depends on DT_HAS_REALTEK_AMEBA_GPIO_ENABLED
+	help
+	  Enable GPIO controller driver for Realtek Ameba series SoC
+
+config GPIO_DEBOUNCE_EN
+	bool "Ameba GPIO Interrupt Debounce Enable"
+	depends on GPIO_AMEBA
+	default y
+	help
+	  When Enable GPIO Interrupt Debounce, the external signal can be debounced to remove any spurious glitches that are less than one period(about 32us) of the external debouncing clock.
```

Comment:
maximum length of 100 characters, wrap lines (1x tab, followed by 2x spaces for each help text line)